CHANGELOG.md

```diff
@@ -6,14 +6,20 @@ documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [6.2.0] - 2022-01-24
+
+- Support PHP 8.1
```

Comment:
\* 8.2

Reply:
This was a rebase mistake, this change is from the "prepare 6.2.0 release" commit.